app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -160,7 +161,7 @@ module.exports = React.createClass
           disabled = @getDisabledAttribute(i)
           selectedOption = if @state.selectedOptions[i]?.value then @state.selectedOptions[i] else null
           optionsKeys.push selectedOption.value if selectedOption?.value
-          <div key={select.id}>
+          <div id={select.id} key={select.id}>
```

Comment:
I thought so as well, but can't get it to work.
`wrapper.find('div').find('[id="countyID"]')` works, but
`wrapper.find('div').find('[key="countyID"]')` doesn't work.